src/components/ShowerInfo.js

```diff
@@ -0,0 +1,30 @@
+import '../styles/ShowerInfo.css'
+
+const ShowerInfo = () => {
+  return (
+    <section className='info-container'>
+      <div className='shower-info-text'>
+        <h1 className='shower-info-title'>Need Help Shortening Your Time?</h1>
+        <ul className='link-list'>
+          <li className='link'>some link or tip</li>
+          <li className='link'>some link or tip</li>
+          <li className='link'>some link or tip</li>
+        </ul>
+      </div>
+      <div className='shower-data'>
+        <div className='shower-data-text-box'>
+          <h3 className='shower-data-text'>Average</h3>
+          <h3 className='shower-data-text'>Today</h3>
+          <h3 className='shower-data-text'>Yesterday</h3>
+        </div>
+        <div className='shower-data-num-box'>
+          <h3 className='shower-data-num'>6:19</h3>
```

Comment:
Thank you for getting this structure set up! I'm excited to see how to incorporate a functional timer.